engine/class_modules/paladin/sc_paladin_protection.cpp

```diff
@@ -439,6 +448,7 @@ void shield_of_the_righteous_buff_t::expire_override( int expiration_stacks, tim
 
 struct shield_of_the_righteous_t : public holy_power_consumer_t<paladin_melee_attack_t>
 {
+  double punish_the_guilty_value;
```

Comment:
Is it really necessary to use a new value inside sotr instead of just calling p -> conduit.punish_the_guilty.percent() during runtime?

Reply:
Was in response to https://github.com/simulationcraft/simc/pull/5386#discussion_r501386063 but simplifying it appeals to me. I suppose in comparison to that earlier review the value should already be loaded up in a variable rather than having to look it up.